AMR Curation 
--------------------------------------------

**Actively monitored by the CARD Curation Team.**

A number of databases and software tools exist for DNA sequence-based investigation of antimicrobial resistance (see `Boolchandani, D'Souza, & Dantas. 2019. Sequencing-based methods and resources to study antimicrobial resistance. Nat Rev Genet <https://www.nature.com/articles/s41576-019-0108-4>`_). We encourage researchers, software developers, and AMR data curators to use this repository to submit, discuss, and resolve AMR curation issues. Anyone with a GitHub account can submit an issue. AMR software developers or database curators are welcome to join the repository as `Contributors <https://github.com/arpcard/amr_curation/graphs/contributors>`_ to share files, data, & code as well as add documentation (send request to card@mcmaster.ca or `submit a GitHub issue <https://github.com/arpcard/amr_curation/issues>`_).

If you are `submitting a question or correction <https://github.com/arpcard/amr_curation/issues>`_ about AMR data in a particular database, please indicate the database in your submission. If your question or issue is about software, please use the software's own repository or email address to report the problem.

Resources
--------------------------------------------

* `Wikipedia's list of Antimicrobial Resistance databases <https://en.wikipedia.org/wiki/List_of_biological_databases#Antimicrobial_Resistance_databases>`_
* `Google Spreadsheet of AMR Databases and Software <https://docs.google.com/spreadsheets/d/1psHogF0rZIIq9AXbFfjQNYVXoOVHEYPOi9CYLvtO04U/edit?usp=sharing>`_ | *help us keep this up-to-date!*

Participant List
--------------------------------------------

* `Comprehensive Antibiotic Resistance Database <http://card.mcmaster.ca>`_ | @arpcard
* NCBI `Beta-Lactamase Alleles <https://www.ncbi.nlm.nih.gov/bioproject/305729>`_ and `Bacterial Antimicrobial Resistance Reference Gene Database <https://www.ncbi.nlm.nih.gov/bioproject/PRJNA313047>`_
* Public Health Agency of Canada (PHAC)'s `IRIDA AMR Detection Pipeline <https://github.com/phac-nml/irida-plugin-amr-detection>`_ | @apetkau
* `ResFinder & PointFinder <https://cge.cbs.dtu.dk/services/ResFinder/>`_, database and software for identification of acquired antimicrobial resistance genes and/or chromosomal mutations in total or partial sequenced isolates of bacteria
* `ARGDIT <https://github.com/phglab/ARGDIT>`_, a toolkit to validate and consolidate antimicrobial resistance gene databases | @jimmykhchiu
* `AMRTime <https://github.com/beiko-lab/AMRtime>`_, a tool to predict the presence of antimicrobial resistance in clinical metagenomes | @fmaguire
* `MEGARes <https://megares.meglab.org>`_, an Antimicrobial Database for High-Throughput Sequencing
* `AmrPlusPlus software <https://github.com/cdeanj/amrplusplus>`_, a high-throughput analysis pipeline to identify and characterize resistance genes from metagenomic sequence data | @cdeanj
* `AMR++ Shiny <https://github.com/lakinsm/amrplusplus-shiny>`_, statistics interface for metagenomic resistome and microbiome analyses | @lakinsm
* Agriculture and Agri-Food Canada | @ropolomx
